### DISCUSSION TITLE
Can't run steam without steamwebhelper

### DISCUSSION THREAD

**Discussion by IOPPWWT** [Whisky-App/Whisky#621] [Q&A]
I install the steam properly. Then run steam.exe, but I can't start it![image](https://github.com/Whisky-App/Whisky/assets/85209854/eb61b5d3-922d-4f9e-a718-7352b537aa2c)

**Comment by frisk099** [comment]
Try uninstall and then install it.

**Reply by TheBeginnerPro** [reply]
Hi, I have the same problem! Did you solve it?

**Comment by TheBeginnerPro** [comment]
Hi, I have the same problem! Did you solve it?

**Reply by IOPPWWT** [reply]
no.You can try CrossOver instead

**Reply by TheBeginnerPro** [reply]
Thank you. Does cross over work better?

**Reply by akoasatom** [reply]
https://www.reddit.com/r/macgaming/comments/192xrdm/steamwebhelper_stopped_working_in_whisky/
